### ISSUE TITLE
Tasks CalDav support

### ISSUE THREAD

**Issue by Dunrar** [SimpleMobileTools/Simple-Calendar#1742]
At the moment I am trying to get out of the Google ecosystem and really like Simple Calendar Pro in conjunction with Nextcloud + Calendar + Tasks. But for it to really be a viable alternative to Google Calendar it misses a few things for me. Tasks CalDav support is maybe the most important. Would really like to see this. Thanks.

**Comment by lisboarocha** [comment]
Hello,
This function is actually really important, cause without it we need another app to sync tasks.
Do you have any idea if it would be possible and when?
Thank you very much for this app!
Best regards

**Comment by tibbi** [comment]
not yet, cannot promise any date yet.

**Comment by lisboarocha** [comment]
Dear @tibbi,
Thank you for your answer.
It's really sad! 
I'm taking some hours now to put up an alternative and it will not be as good as it would be in this app.
I hope you will be able to make this some time. Me and others will be glad to use the app.
Thank you again for this job.
Best regards

**Comment by lisboarocha** [comment]
Actually, have you thinked about make an specific crowdfunding for an issue like this one? Would it be an option?
As @Dunrar said, this point is an essential one to mae possible to a lot of people change from G.

**Comment by 0xFunKy** [comment]
Crowdfounding sounds like a good option to me. For example with Bountysource. I would participate in the bounty.

For now I am just going to donate because I am a happy user of the Simple Mobile Tools.
Hoping that the issue gets addressed one day

**Comment by tibbi** [comment]
we added task importing and exporting via .ics file recently, so we are getting there. But adding CalDAV support will be a lot more work.

**Comment by Impasse52** [comment]
Hello! Thank you for your hard work. Are there any updates on this? It would make the integration with CalDAV basically perfect.

**Comment by tibbi** [comment]
not really, it hasnt been added yet as tasks work in a whole different way than events and it requires a ton of work to have it working.

**Comment by Impasse52** [comment]
That's unfortunate but I respect your decision. I hope that eventually it'll land on your roadmap, it would be a really sick addition.

**Comment by MarshCastle** [comment]
I'm a +1 for this.
Sad that it's a lot of work, but I guess it is what it is.
Other wise it's a brilliant app!

Would love to see apps self fund via something like Alby - like v4v in podcasts (see https://podcastindex.org).

I think this could help loads with funding issues.

**Comment by bcspragu** [comment]
Chiming in here - Just started using Simple Calendar Pro and would love to have task syncing support. I had originally emailed this, but the `hello@simplemobiletools.com` alias bounced for me. In short: I see that DAVx5 supports syncing tasks, and I want to try adding compatibility for syncing Simple Calendar tasks.

I've been writing Kotlin (but not Android stuff) in my day job recently, so if the team is willing to give me some pointers, I can try and give this a shot. Looking at [the DAVx5 syncing code](https://github.com/bitfireAT/davx5-ose/tree/24d4ba65e512e7cd73daf48da521aab1040f455c/app/src/main/kotlin/at/bitfire/davdroid/sync), nothing seems _too_ crazy.

**Comment by Palaress** [comment]
Hi @bcspragu , I was searching for the issue at the fossify calendar app.

I think SimpleMobileTools is more or less abandoned and been replaced by fossify. For example there where no PRs merged since Nov 2023, and tibbi wasn't active in 2025 at all, and only with just a few activities in 2024.

I think your efforts would be much more useful in the Fossify's Calendar app. 

I'm not linked to them and have been using many of the SimpleMobileTools for years and am just glad its continued in some way. Some of the apps I am not that happy with, tbh, but Calendar and Gallery feel as good as the originals.

Here the corresponding issue: https://github.com/FossifyOrg/Calendar/issues/36.

**Comment by acmpo6ou** [comment]
> Chiming in here - Just started using Simple Calendar Pro and would love to have task syncing support. I had originally emailed this, but the `hello@simplemobiletools.com` alias bounced for me. In short: I see that DAVx5 supports syncing tasks, and I want to try adding compatibility for syncing Simple Calendar tasks.
> 
> I've been writing Kotlin (but not Android stuff) in my day job recently, so if the team is willing to give me some pointers, I can try and give this a shot. Looking at [the DAVx5 syncing code](https://github.com/bitfireAT/davx5-ose/tree/24d4ba65e512e7cd73daf48da521aab1040f455c/app/src/main/kotlin/at/bitfire/davdroid/sync), nothing seems _too_ crazy.

Are you still using SimpleMobileTools? Check out this thread https://github.com/SimpleMobileTools/General-Discussion/issues/241#issue-2020627780 some shady stuff is going with them. Better check out Fossify, they forked SimpleMobileTools and keep them truly open source.

**Comment by MarshCastle** [comment]
I've been using the android 'tasks' app + davx5.

It syncs to fossify calendar fine, and I get lots of tadk meta in there too.

Martin Castle

14 Aug 2025 19:41:52 astroboy ***@***.***>:

**Comment by bcspragu** [comment]
> Are you still using SimpleMobileTools? Check out this thread [SimpleMobileTools/General-Discussion#241 (comment)](https://github.com/SimpleMobileTools/General-Discussion/issues/241#issue-2020627780) some shady stuff is going with them. Better check out Fossify, they forked SimpleMobileTools and keep them truly open source.

Thanks for the heads up, not sure how I missed that, will follow up over there
